### PR TITLE
added ethereumAddress

### PR DESCRIPTION
**Pull request by awoie**
added ethereumAddress

### CODE REVIEW

**Comment by OR13** [comment]
Looks great!

**Comment by Therecanbeonlyone1969** [comment]
Agree! Looks great!

**Comment by oed** [comment]
It's probably a good idea to add a checksum to the address using [EIP55](https://eips.ethereum.org/EIPS/eip-55).

**Comment by wyc** [comment]
+1!

Elaborating on @peacekeeper's note for any onlookers; you can only recover the public key associated with an Ethereum address if you have a signature it produced, such as in a signed Ethereum transaction.

File: index.html [hunk 1]
```diff
@@ -899,6 +899,37 @@ <h3>cipherKey</h3>
...
+          <dt>Range</dt>
+          <dd>xsd:string</dd>
+        </dl>
+        <p>
+    The following example demonstrates the expression of an Ethereum address in hex format.
+        </p>
+        <pre class="example prettyprint language-jsonld">
+{
+    "id": "did:example:123#ethereumAddress",
+    "type": "EcdsaSecp256k1RecoverySignature2020",
```
Comment (rhiaro) [inline index.html:927]: Can you include/mention/link to the context where `EcdsaSecp256k1RecoverySignature2020` is defined in/with this example for completeness?

Comment (OR13) [reply index.html:927]: I agree, but let's tackle this in a subsequent PR. I have opened: https://github.com/w3c-ccg/security-vocab/issues/43

**Comment by OR13** [comment]
7 Days open, many approvals, no objections, merging.

**Comment by OR13** [comment]
Thanks to everyone who reviewed this PR, left comments, now please continue the discussion on support for ethereum and bitcoin in the did core context here: https://github.com/w3c/did-spec-registries/pull/73

**Pull request merged**